### ISSUE TITLE
[Query] Specify outer license with project.

### ISSUE THREAD

**Issue by e0lithic** [dashingsoft/pyarmor#337]
Hi,

When using .pyarmor_config using project. Is there a way to specify "outer license". Currently the "license_file" takes the license file as input and embeds it. 

Thanks.

**Comment by jondy** [comment]
Config it with special value `outer`
```
pyarmor config --with-license outer
```

**Comment by e0lithic** [comment]
I am getting the following error when I try to build using "outer".
```python
{
  "version": "2.0",
  "name": "backend",
  "title": "backend",
  "src": "src",
  "is_package": null,
  "manifest": "global-include *.py",
  "entry": "manage.py",
  "output": "pkg",
  "runtime_path": null,
  "restrict_mode": 1,
  "obf_code": 2,
  "obf_mod": 2,
  "wrap_mode": 1,
  "advanced_mode": 2,
  "bootstrap_code": 1,
  "cross_protection": 1,
  "plugins": null,
  "platform": null,
  "package_runtime": 1,
  "enable_suffix": 0,
  "license_file": "outer",
  "build_time": 0.0
}
```

Output 
```
INFO     PyArmor Version 6.4.2
ERROR    The output path can not be same as src in the project
```

The configuration certainly has the src and output as different paths.

Thanks

**Comment by jondy** [comment]
Would you please run it with -d?
```
pyarmor -d build
```

**Comment by e0lithic** [comment]
Here's the log
```log
DEBUG    PyArmor installation path: /home/xyz/.pyenv/versions/3.7.6/envs/test/lib/python3.7/site-packages/pyarmor
DEBUG    PyArmor home path: /home/xyz/.pyarmor
DEBUG    Native platform is linux.x86_64
DEBUG    Search dynamic library in the path: /home/xyz/.pyenv/versions/3.7.6/envs/test/lib/python3.7/site-packages/pyarmor/platforms
DEBUG    Loaded dynamic library: /home/xyz/.pyenv/versions/3.7.6/envs/test/lib/python3.7/site-packages/pyarmor/platforms/linux/x86_64/_pytransform.so
DEBUG    The version of core library is (34, 8, 7)
INFO     PyArmor Version 6.4.2
Traceback (most recent call last):
  File "/home/xyz/.pyenv/versions/test/bin/pyarmor", line 10, in <module>
    sys.exit(main_entry())
  File "/home/xyz/.pyenv/versions/3.7.6/envs/test/lib/python3.7/site-packages/pyarmor/pyarmor.py", line 1447, in main_entry
    main(sys.argv[1:])
  File "/home/xyz/.pyenv/versions/3.7.6/envs/test/lib/python3.7/site-packages/pyarmor/pyarmor.py", line 1439, in main
    args.func(args)
  File "/home/xyz/.pyenv/versions/3.7.6/envs/test/lib/python3.7/site-packages/pyarmor/pyarmor.py", line 198, in _build
    project.open(args.project)
  File "/home/xyz/.pyenv/versions/3.7.6/envs/test/lib/python3.7/site-packages/pyarmor/project.py", line 147, in open
    self._load(filename)
  File "/home/xyz/.pyenv/versions/3.7.6/envs/test/lib/python3.7/site-packages/pyarmor/project.py", line 138, in _load
    self._check(os.path.dirname(filename))
  File "/home/xyz/.pyenv/versions/3.7.6/envs/test/lib/python3.7/site-packages/pyarmor/project.py", line 128, in _check
    'The output path can not be same as src in the project'
AssertionError: The output path can not be same as src in the project
(test) 
```

**Comment by jondy** [comment]
It's a bug. I'll fix it in next version. As a quick workaround, just edit `.pyarmor_config`, set `license_file` to `null`, then run `build` with `--with-license outer`:
```
pyarmor build --with-license outer
```

**Comment by jondy** [comment]
Fixed in [v6.4.3](https://github.com/dashingsoft/pyarmor/releases/tag/v6.4.3)
